### PR TITLE
[stable/rocketchat]  Update README with HA configuration info

### PR DESCRIPTION
**Pull request by Sing-Li**
## What this PR does / why we need it:
From recent chart updates, the README has become outdated with respect to the chart's capabilities.

This PR brings the description and instructions up to date.

#### Special notes for your reviewer:

Please review and merge @geekgonecrazy ;  thanks.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/rocketchat/README.md [hunk 1]
```diff
@@ -13,12 +13,13 @@ $ helm install stable/rocketchat --set mongodb.mongodbPassword=$(echo -n $(opens
...
 This chart bootstraps a [Rocket.Chat](https://rocket.chat/) Deployment on a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager. It provisions a fully featured Rocket.Chat installation.
-For more information on Rocket.Chat and its capabilities, see its [documentation](https://rocket.chat/docs/).
+
+In addition, this chart supports scaling of Rocket.Chat for increased server capacity and high availability.  For more information on Rocket.Chat and its capabilities, see its [documentation](https://rocket.chat/docs/).
 
 ## Prerequisites Details
 
 The chart has an optional dependency on the [MongoDB](https://github.com/kubernetes/charts/tree/master/stable/mongodb) chart.
-By default, the MongoDB chart requires PV support on underlying infrastructure (may be disabled).
+By default, the MongoDB chart requires StatefulSet support on underlying infrastructure (may be overridden).
```
Comment (geekgonecrazy) [inline stable/rocketchat/README.md]: Might be worth keeping PV.. as some k8s setups might not have configured a provider for provisioning volumes

**Review by geekgonecrazy — CHANGES_REQUESTED** [review]

File: stable/rocketchat/README.md (hunk 1 above)
Comment (Sing-Li) [reply stable/rocketchat/README.md]: Reverted to retain PV.  Thanks.

**Comment by Sing-Li** [comment]
fixed the DCO with the last commit

**Pull request merged**